lms/static/sass/elements/_icons.scss

```diff
@@ -1,11 +1,22 @@
-.verified-cert-icon {
+.certificate-icon svg {
   @include float(left);
   @include margin-right($baseline*0.4);
 
-  margin-top: ($baseline/4);
-  background: url('#{$static-path}/images/icon-sm-verified.png') no-repeat;
-  background-position: -4px -5px;
+  margin-top: ($baseline/10);
+  width: 23px;
+  height: 20px;
+  padding: 2px;
   background-color: palette(grayscale, white-t);
-  width: 20px;
-  height: 18px;
+  border-style: solid;
+  border-width: 2px;
+}
+
+.green-certificate-icon svg {
+  fill: palette(success, dark);
+  border-color: palette(success, dark);
+}
+
+.blue-certificate-icon svg {
```

Comment:
Nitpick: I would prefer that a class is added to the svg element and that is referenced instead of the svg element itself.